server/Startup.cs

```diff
@@ -32,6 +33,19 @@ public void ConfigureServices(IServiceCollection services)
       ));
       services.AddTransient<IGameService, GameService>();
       services.AddControllers();
+
+      services.AddSignalR(options => options.EnableDetailedErrors = true);
+
+      services.AddCors(options =>
+      {
+        options.AddPolicy("default", policy =>
+        {
+          policy.WithOrigins("http://localhost:3000")
+                .AllowAnyHeader()
+                .AllowAnyMethod()
+                .AllowCredentials();
+        });
+      });
```

Comment:
not needed if we use the react dev server proxy

Reply:
But that only works for dev right ? This is needed imo, just need to add a config instead of hard coding the url.

Reply:
Well, whatever solution we choose to host our front, we can configure it to proxy request to the API server (and thus removing the need for CORS headers since from the browser POV, the request has been made to the same domain)